src/shared/components/oncoprint/controls/ConfirmNgchmModal.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+
+
+export default class ConfirmNgchmModal extends React.Component {
+
+    state: {show: boolean, callback: () => void } = {
```

Comment:
for consistency sake, lets use mobx observable props to handle state instead of react state.  so, the component gets @observer decorator and show and callback are properties that have @observable decorator.   
There are many devs working on the project and I don't want them getting inconsistent message.

Reply:
Done, I think.  react state is gone.  (Still in RC)